packages/components/src/core/DialogTitle/style.ts

```diff
@@ -24,7 +24,11 @@ export const StyledDialogTitle = styled(DialogTitle)`
 export const Title = styled(Typography)`
   ${fontHeaderXl}
 
-  color: black;
+  ${(props) => {
+    return `
+      color: ${props.theme.palette.text.primary};
```

Comment:
Just thinking about this a bit... `Typography` should be theme-aware. I think we might be able to get away with removing `color` entirely!

Reply:
Thanks for paving the way for our components to be runtime theme aware 💡 !! 

Yeah using semantic `text.primary` sounds like best practice to me too 👍 !

Reply:
Cool, going to remove the color entirely for `Title` then. Keeping for `SubTitle` since that won't be a secondary color by default!